allure-jsonunit/src/main/resources/tpl/diff.ftl

```diff
@@ -0,0 +1,311 @@
+<html>
+<head>
+    <meta http-equiv = "content-type" content = "text/html; charset=UTF-8">
+    <script type='text/javascript' src="https://cdn.jsdelivr.net/npm/jsondiffpatch/dist/jsondiffpatch.umd.min.js" charset="utf-8"></script>
```

Comment:
lets include minified version directly to template in order to make it work offline